### PR TITLE
[PUB-1920] Add functionality to open and close Story Preview

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
Add functionality to open and close Story Preview.

## Context & Notes
[PUB-1920](https://buffer.atlassian.net/browse/PUB-1920)

Next PR: Adapt Preview to use data being passed by these changes and make carousel work.

## Screenshots (if appropriate)
https://hi.buffer.com/1f8ed4e2653b

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/story-preview/components/PreviewPopover/index.jsx [hunk 1]
```diff
@@ -2,19 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { white } from '@bufferapp/ui/style/colors';
-import { Popover } from '@bufferapp/components';
+import { LightboxCarousel } from '@bufferapp/publish-shared-components';
```
Comment (albennett) [inline packages/story-preview/components/PreviewPopover/index.jsx]: Just checked out the Lightbox Carousel component. Great job with refactoring to include the arrows 💯

**Comment by albennett** [comment]
Also, I think it's great that you're breaking tasks out into smaller PRs 💯

**Comment by anafilipadealmeida** [comment]
> Also, I think it's great that you're breaking tasks out into smaller PRs 💯

Smaller PRs, easier to catch mistakes and review 🙂 Also makes tasks feel less daunting 😅
Thanks for reminding me about the Composer! 🙌 (Will take care of connecting the Preview there in the end, at this point I just needed data to make the carousel work 😂)

**Comment by MayaUribe** [comment]
Late for this one! Thank you for the work here @anafilipadealmeida 🙌🚀

**Pull request merged**